src/ansys/dpf/composites/layup_info/material_operators.py

```diff
@@ -23,17 +23,16 @@ class MaterialOperators:
         evaluate material properties. This container cannot be
         queried in Python, but it can be passed to other DPF operators
         that evaluate the properties.
-    result_info_provider:
```

Comment:
AFAICT removing this parameter is a backwards-incompatible change. Shouldn't we deprecate it first?